src/quo2/components/info/page_nav.cljs

```diff
@@ -0,0 +1,242 @@
+           left-section-icon-background-color
+           right-section-icons]}]
+  (let [{:keys [height width]}      (dimensions/window)
+        put-middle-section-on-left? (or align-mid?
+                                        (> (count right-section-icons) 1))
+        mid-section-props {:mid-section-type                 mid-section-type
+                           :horizontal-description?          horizontal-description?
+                           :mid-section-main-text            mid-section-main-text
+                           :one-icon-align-left?             one-icon-align-left?
+                           :mid-section-right-icon           mid-section-right-icon
+                           :mid-section-icon                 mid-section-icon
+                           :mid-section-main-text-icon-color mid-section-main-text-icon-color
+                           :mid-section-left-icon            mid-section-left-icon}]
+    [rn/view {:style (cond->
+                      {:display            :flex
+                       :flex-direction     :row
+                       :width              width
+                       :height             (* 0.0497 height)
+                       :align-items        :center
+                       :padding-horizontal 8
+                       :justify-content    :space-between}
+                       page-nav-background-uri (assoc :background-color page-nav-color)
+                       page-nav-color (assoc :background page-nav-background-uri))}
+     [rn/view {:style {:flex           1
+                       :flex-direction :row
+                       :align-items    :center}}
+      [rn/view (merge
+                icon-styles
+                {:background-color left-section-icon-background-color
+                 :width            32
+                 :height           32}
+                (when put-middle-section-on-left? {:margin-right 5}))
+       [icons/icon left-section-icon (icon-props left-section-icon-color :big)]]
+      (when put-middle-section-on-left?
+        [mid-section (merge {:left-align?                       true
+                             :mid-section-description           mid-section-description
+                             :mid-section-description-color     mid-section-description-color
+                             :mid-section-description-icon      mid-section-description-icon
+                             :align-mid?                        align-mid?
+                             :mid-section-description-user-icon mid-section-description-user-icon}
+                            mid-section-props)])]
+     (when-not put-middle-section-on-left?
+       [mid-section mid-section-props])
+     [rn/view  (update-in
+                centrify
+                [:style]
+                #(assoc % :flex-direction :row
+                        :flex           1
+                        :justify-content :flex-end))
+      (let [last-icon-index (- (count right-section-icons) 1)]
```

Comment:
you could just assign this prop to the last element

Reply:
Done

Reply:
do see the changes